src/pages/lists/action-list/action-list-local-actions/scorecard.tsx

```diff
@@ -0,0 +1,90 @@
+import * as Colors from '@brightlayer-ui/colors';
+import { Hero, HeroBanner, ScoreCard } from '@brightlayer-ui/react-components';
+import { GradeA, Temp, Moisture as Humidity } from '@brightlayer-ui/icons-mui';
+import Button from '@material-ui/core/Button';
+import { createStyles, makeStyles, Theme, useTheme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        scorecardRoot: {
+            width: 350,
+            height: 250,
+            marginTop: 16,
+            marginLeft: 16,
+        },
+        scorecardHeader: {
+            display: 'flex',
+            height: 48,
+            borderBottom: `1px solid ${theme.palette.divider}`,
+        },
+        scorecardHeaderContent: {
+            padding: '0 16px',
+            alignItems: 'center',
+        },
+        bodyWrapper: {
+            flex: 0,
+        },
+        badgeWrapper: {
+            flex: 1,
+        },
+    })
+);
+
+export const LocalActionsScoreCard = (): JSX.Element => {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+
+    return (
+        <ScoreCard
+            classes={{
+                root: classes.scorecardRoot,
+                header: classes.scorecardHeader,
+                headerContent: classes.scorecardHeaderContent,
+                bodyWrapper: classes.bodyWrapper,
+                badgeWrapper: classes.badgeWrapper,
+            }}
+            headerTitle={'Overview'}
+            headerColor={Colors.white[50]}
+            headerFontColor={Colors.blue[500]}
+            badge={
+                <HeroBanner style={{ minWidth: 210 }}>
+                    <Hero
+                        icon={<Humidity fontSize={'inherit'} htmlColor={Colors.blue[200]} />}
+                        label={'Humidity'}
+                        value={54}
+                        units={'%'}
+                        iconSize={48}
+                        fontSize={'normal'}
+                    />
+                    <Hero
+                        icon={<Temp fontSize={'inherit'} htmlColor={Colors.red[500]} />}
+                        label={'Temperature'}
+                        iconSize={48}
+                        value={97}
+                        units={'°F'}
+                        fontSize={'normal'}
+                    />
+                    <Hero
+                        icon={<GradeA fontSize={'inherit'} htmlColor={Colors.green[500]} />}
+                        label={'Overall'}
+                        value={96}
+                        units={'/100'}
+                        iconSize={48}
+                        fontSize={'normal'}
+                    />
+                </HeroBanner>
+            }
+            actionRow={
+                <Button
+                    onClick={(): void => {}}
+                    variant={'contained'}
+                    color={'primary'}
+                    style={{ margin: 16, width: 'calc(100% - 32px)' }}
```

Comment:
In the spirit of avoiding magic numbers, always use `theme.spacing(2)` instead of  `16`. 

Also, I find the `width` to be an abuse of `calc` function, as it tends to lower the code readability.